README.md

```diff
@@ -1,4 +1,4 @@
-![Kubernetes Cloud Provider for Equinix Metal](docs/images/github_kubernetes-cloud-provider_share.jpeg)
+<h1 align="center"><img src=docs/images/github_kubernetes-cloud-provider_share.jpeg></h1>
```

Comment:
Is this conventional? I don't see a reason to convert the image to HTML rather than markdown. Both appear to render identically in GH.

- We lose the text description of the image
- A heading is defined with no text (alternative to `#` syntax)
- src is not quoted
- align is not an HTML5 supported attributed, so I'm really not sure what it means in the context of a Markdown viewer

Reply:
This is what was recommended in the markdownlint documentation for github projects that typically begin their readme file with an image. So it seemed pretty relevant to us.

Reply:
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading

Reply:
(that linter rule seems silly to me. an alternative could be to move the `#` line about the image)